Package.swift

```diff
@@ -243,8 +243,18 @@ let package = Package(
       publicHeadersPath: ".",
       cSettings: [
         .headerSearchPath("./"),
+      ],
+      linkerSettings: [
+        .linkedFramework("CoreFoundation"),
```

Comment:
Read the docs but I don't really understand what a Linker Setting is and why we are adding CoreFoundation as a linked framework.

Reply:
Usually Xcode detects the need and automatically adds CoreFoundation implicitly, but it wasn't doing so when I ran the new test locally and had undefined symbols - perhaps because the references come from C++.

Reply:
that makes sense, thanks!